app/models/search/indexer.rb

```diff
@@ -28,7 +29,8 @@ def self.all
       BookmarkIndexer,
       PseudIndexer,
       TagIndexer,
-      WorkIndexer
+      WorkIndexer,
+      CollectionIndexer
```

Comment:
It looks like the existing list is aiming for alphabetical order. Can we scoot this one up between `BookmarkIndexer` and `PseudIndexer`?